src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -765,6 +765,17 @@ MavlinkReceiver::handle_message_distance_sensor(mavlink_message_t *msg)
 	// quality value. Also it comes normalised between 1 and 100 while the uORB
 	// signal quality is normalised between 0 and 100.
 	ds.signal_quality = dist_sensor.signal_quality == 0 ? -1 : 100 * (dist_sensor.signal_quality - 1) / 99;
+        qt = const_cast<float*>(ds.q);
+	//mavlink clients
+        mavlink_msg_distance_sensor_send(MAVLINK_COMM_0, ds.timestamp, ds.min_distance, ds.max_distance, ds.current_distance, ds.type, ds.device_id, ds.orientation, ds.variance, ds.h_fov, ds.v_fov, qt, ds.signal_quality);
```

Comment:
Crossing mavlink channels (owned by separate mavlink instances) doesn't look quite right, could you explain what you're trying to accomplish here?

Reply:
I'll be honest, I was troubleshooting mavlink messages issues. Turns out the sensor itself didn't like 5 volts and would refuse to work when connected to a LiPO, but would work perfectly fine in USB. Thought it was a telemetry vs usb issue. Can remove, unnecessary.

Reply:
Please remove yes.